packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -118,7 +118,7 @@ const query = graphql`
         }
       }
       organizations {
-        aiTemplate: featureFlag(featureName: "aiTemplate")
```

Comment:
Help me understand this change here, why would someone request a scope of anything other than Organization in this example?  Couldn't it be inferred since this is a field under Organization?